nautobot/core/tests/test_views.py

```diff
@@ -399,6 +402,48 @@ def test_api_docs_403_unauthenticated(self):
             response = self.client.get(url)
             self.assertHttpStatus(response, 403)
 
+    @load_event_broker_override_settings(
+        EVENT_BROKERS={
+            "SyslogEventBroker": {
+                "CLASS": "nautobot.core.events.SyslogEventBroker",
+                "TOPICS": {
+                    "INCLUDE": ["*"],
+                },
+            }
+        }
+    )
+    def test_login_logout(self):
+        self.client.logout()
+        self.user.set_password("pass")
+        self.user.save()
+
+        with self.assertLogs("nautobot.events") as cm:
+            self.client.post(
+                reverse("login"),
+                data={
+                    "username": self.user.username,
+                    "password": "pass",
+                },
+            )
+        self.user.refresh_from_db()
+        payload = {"data": serialize_object_v2(self.user)}
+        self.assertEqual(
+            cm.output,
+            [f"INFO:nautobot.events.nautobot.users.user.login:{json.dumps(payload, indent=4)}"],
+        )
```

Comment:
Maybe add an `assertNotIn("password", cm.output)` and/or `assertNotIn("pass", cm.output)` as well?